aws/table_aws_route53_vpc_association_authorization.go

```diff
@@ -0,0 +1,144 @@
+package aws
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/route53"
+
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+)
+
+func tableAwsRoute53VPCAssociationAuthorization(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_route53_vpc_association_authorization",
+		Description: "AWS Route53 VPC Association Authorization",
+		List: &plugin.ListConfig{
+			Hydrate: listVPCAssociationAuthorization,
+			Tags:    map[string]string{"service": "route53", "action": "ListVPCAssociationAuthorizations"},
+			KeyColumns: plugin.KeyColumnSlice{
+				{
+					Name:    "hosted_zone_id",
+					Require: plugin.Required,
+				},
+			},
+		},
+		Columns: awsGlobalRegionColumns([]*plugin.Column{
+			{
+				Name:        "hosted_zone_id",
+				Description: "The ID of the hosted zone for which you want a list of VPCs that can be associated with the hosted zone.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "vpc_id",
+				Description: "(Private hosted zones only) The ID of an Amazon VPC.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("VPCId"),
+			},
+			{
+				Name:        "vpc_region",
+				Description: "(Private hosted zones only) The region that an Amazon VPC was created in.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("VPCRegion"),
+			},
+
+			// Steampipe standard columns
+			{
+				Name:        "title",
+				Description: resourceInterfaceDescription("title"),
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("HostedZoneId", "VPCId"),
+			},
+			// no tags for this resource as per aws docs
+			{
+				Name:        "akas",
```

Comment:
Does AWS recognize this as a resource type, or provide an ARN format for it? If so, we should retain the 'aks' column; otherwise, we can remove it.